### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by sonnh-uit**
I add a short guide to build project from source. I was add guide for change signature used create connect to S3 server.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,8 +4,26 @@ S3 benchmarking tool.
...
+Warp require minimum version is `go1.22`, please ensure you have compatible version for this build. 
+
+You can follow easy step below to build project
+- Clone project
+```
+git clone https://github.com/minio/warp.git
+```
+- Change directory and build
+```
+cd warp && go run main.go
```
Comment (klauspost) [inline README.md:21]:
```suggestion
cd warp && go build
```

File: README.md [hunk 2]
```diff
@@ -4,8 +4,26 @@ S3 benchmarking tool.
...
+```
+- To run a test, please run
+```
+go run main.go command [options]
```
Comment (klauspost) [inline README.md:25]:
```suggestion
./warp [options]
```

File: README.md [hunk 3]
```diff
@@ -27,6 +45,8 @@ you can enable [server-side-encryption](https://docs.aws.amazon.com/AmazonS3/lat
 of objects using `--encrypt`. A random key will be generated and used for objects.
 To use [SSE-S3](https://docs.aws.amazon.com/AmazonS3/latest/userguide/UsingServerSideEncryption.html) encryption use the `--sse-s3-encrypt` flag.
 
+`S3V4` is default signature to create connect to S3 server. If it not same yours, change it to `S3V2` by `--signature` flag.
```
Comment (klauspost) [inline README.md:48]:
```suggestion
If your server is incompatible with [AWS v4 signatures](https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html) the older v2 signatures can be used with `--signature=S3V2`.
```

**Review by klauspost — CHANGES_REQUESTED** [review]
Let's build a binary.

**Review by sonnh-uit — COMMENTED** [review]
I was change base on your suggestion.

**Review by klauspost — APPROVED** [review]

**Comment by klauspost** [comment]
Seems like latest Go isn't updated upstream. No biggie.

**Pull request merged**